.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test
```

Comment:
Possibly we will include tests as part of the build step (my argument). If we end up different, we must also run the tests pipeline on the main branch for coverage reports to function